PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -4,12 +4,20 @@ This document provides information on how to get your repository set up to take
 
 ## Account Setup
 
-First you will need a Router account, and to configure it for SWORDv2 integration.  See your account page in the 
-router for more information.  You need to provide the following:
-
-1. A username and password for a user in your repository who has the rights to create content via SWORDv2
-2. A collection URL into which the system will deposit the content
-3. Your preferred packaging format for zip files being deposited.  By default (and the only option in this first version) this will be http://purl.org/net/sword/package/SimpleZip
+First you will need a Router account, and set up the SWORDv2 credentials in the 'Manage connection settings' section on your account.  You need to provide the following:
+
+* The repository URL
+* Select a Repository Configuration (Eprints Native, Eprints RIOXX, DSpace native)
+* Select an Eprints deposit location if required (Manage Deposit, Review Queue)
+* A username and password for a user in your repository who has the rights to create content via SWORDv2.
+   * For deposits in Manage Deposit on Eprints - User role   
+   * For deposits in Review Queue on Eprints - Editor or Admin role
+   * For deposits on DSpace - Admin role on the collection 
```

Comment:
In the preceding 3 lines, it would be clearer to mention the repository type earlier, e.g. for first line:
`For Eprints deposits into Manage Deposits queue a User role is required`